### DISCUSSION TITLE
Any way to cease the "Cables is paused"

### DISCUSSION THREAD

**Discussion by archgary** [cables-gl/cables_docs#463] [Answers & Questions] [answered]
Hi all,

Does anyone has any idea/way/method to not get the "Cables is paused"?
The reason I am asking is due to the length of the piece I am working on, and sometimes this comes in the middle of the patch running and I would like to avoid this happening. 
Any suggestions would be much appreciated. 

Thank you!

Best,
archgary

**Comment by pandrr** [comment, marked as answer]
![Screenshot 2023-01-23 at 15 51 46](https://user-images.githubusercontent.com/98792/214070068-43516a59-b774-4b40-836b-50c7b4d45006.png)

you can turn it off in the editor preferences

**Reply by archgary** [reply]
Thank you! @pandrr, this is useful!
